FreeAPS/Sources/Modules/Base/BaseProvider.swift

```diff
@@ -11,6 +11,7 @@ class BaseProvider: Provider, Injectable {
     @Injected() var deviceManager: DeviceDataManager!
     @Injected() var storage: FileStorage!
     @Injected() var bluetoothProvider: BluetoothStateManager!
+    @Injected() var contactTrickManager: ContactTrickManager!
```

Comment:
not sure if I should have added this here - it's referenced from the `ContactTrick.Provider` when contacts are saved